rest/model/build.go

```diff
@@ -138,6 +139,7 @@ type APITaskCache struct {
 	StatusDetails   apimodels.TaskEndDetail `json:"task_end_details"`
 	StartTime       *time.Time              `json:"start_time"`
 	TimeTaken       time.Duration           `json:"time_taken"`
+	TimeTakenMs     APIDuration             `json:"time_taken_ms"`
```

Comment:
nit: TimeTakenMS; we should capitalize all of an acronym

Reply:
Fixed!